builder/tencentcloud/cvm/image_config.go

```diff
@@ -32,7 +32,7 @@ type TencentCloudImageConfig struct {
 	// after your image created.
 	ImageShareAccounts []string `mapstructure:"image_share_accounts" required:"false"`
 	// Do not check region and zone when validate.
-	SkipValidation bool `mapstructure:"skip_region_validation" required:"false"`
+	ImageSkipValidation bool `mapstructure:"image_skip_region_validation" required:"false"`
```

Comment:
I see what happened here. This skip_region_validation is essentially the same as the one in Access Config, as you see before the validRegion function is in Access config. This field was probably copied just to support the check. Instead of renaming I would suggest adding a new config type struct in access_config.go just for the SkipValidation field that can then be embedded into both TencentCloudImageConfig  and TencentCloudAccessConfig

Something like
```go

type TenencentCloudRegionValidation struct {
	// Do not check region and zone when validate.
	SkipValidation bool `mapstructure:"skip_region_validation" required:"false"`
}

```

This will ensure no breaking change and will work as intended.

Reply:
Unfortunately I don't think we can do what you suggest as it will still imply a duplicate argument unfortunately.
I ended-up creating a single boolean in the parent config, and propagate its value to private fields on the substructures that needed it.

Reply:
Hmm.. this is interesting if this is the case I suspect other plugins that use this embedded pattern would have this issue. Just in case I'm recommending we do something like https://github.com/hashicorp/packer-plugin-amazon/blob/2efec668b60523b60c43307e4b4a17b1d6328237/builder/common/run_config.go#L77 which is then embedded into each of the builders https://github.com/hashicorp/packer-plugin-amazon/blob/2efec668b60523b60c43307e4b4a17b1d6328237/builder/ebs/builder.go#L37

Reply:
With this approach I fear we'll experience the same issue since a single builder has substructures that use this attribute, so if we make a shared structure and embed it in both config structs, it'll still be a conflict, hence the approach I took to propagate the value from an upper level to the sub-structures for later validation.
We could make a `Meta` structure to define such attributes later, but I would think we still have to manually propagate the values in this case.

Reply:
Ah I see - this would be embedded twice within the same config structure for the cvm builder. For whatever reason I thought this was a config being shared across to builders. Sorry for not looking at the builder config entirely. Yeah this makes sense to me.